### ISSUE TITLE
Installing from the Snap Store makes the application awkward

### ISSUE THREAD

**Issue by lonnyshoal** [bluescan/tacentview#187]
I first installed this application from the Snap Store and found this application is not good because it takes a long time to start and cannot access files from the removable storage. And I uninstalled and reinstall this application with the Deb file. Now this application starts fast and of course no issue about accessing files from the removable storage.

**Comment by bluescan** [comment]
I think the snap may need special permission to have the snap access removable media... will leave issue open for now until I find out. As for startup times, that'll probably have more to do with the snap VM rather than the app itself... so may very well be outside of my control.

**Comment by the51stater** [comment]
Yes, both Snap and Flatpak utilize container-like sandboxing. I believe any app will have delay running from any container.
As for the permission for accessing removable media, I have known how to do it. With both Snap and Flatpak, the user needs to manually configure permissions for each app.
BTW, I'm happy with Tacent View's DEB build.
